### PR TITLE
chore: fix text editor widget cypress spec

### PR DESCRIPTION
**Pull request by sbalaji1192**
## Automation

/ok-to-test tags="@tag.TextEditor"

### :mag: Cypress test results

> [!IMPORTANT]
> Workflow run: <https://github.com/appsmithorg/appsmith/actions/runs/8504305878>
> Commit: `277169acb8b7b5b9ef9903159fe7ac40bd04e64a`
> Cypress dashboard url: <a href="https://internal.appsmith.com/app/cypress-dashboard/rundetails-65890b3c81d7400d08fa9ee5?branch=master&workflowId=8504305878&attempt=1" target="_blank">Click here!</a>
> All cypress tests have passed 🎉🎉🎉

## Summary by CodeRabbit

- **Tests**
	- Updated test descriptions and added a delay for improved reliability in the RichTextEditor widget within the Form Widget.

### CODE REVIEW

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/Form/FormWidget_With_RichTextEditor_spec.js [hunk 1]
```diff
@@ -22,6 +22,8 @@ describe(
         widgetsPage.widgetNameSpan,
       );
 
+      cy.wait(1000);
```
Comment (mohanarpit) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/Form/FormWidget_With_RichTextEditor_spec.js:25]: @sbalaji1192 Any reason we are adding a static wait of 1 sec here? Why does the test pass with the static wait but fails without it?

**Review by somangshu — CHANGES_REQUESTED** [review]
Some review comment already exists

**Review by somangshu — APPROVED** [review]
LGTM

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/Form/FormWidget_With_RichTextEditor_spec.js (hunk 1 above)
Comment (sbalaji1192) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/Form/FormWidget_With_RichTextEditor_spec.js]: cypress is not waiting for the elements inside the iframe to have the expected text. which it does for the elements present in the main frame. So added wait for the change to get reflected on the iframe.

**Pull request merged**